docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
@jkahrs I know, but it's not even in `stable` repository, which still hold 17.03.1